docs/index.rst

```diff
@@ -44,15 +44,20 @@ Small sample script that shows how to get started with RedisCluster. It can also
     >>> # Requires at least one node for cluster discovery. Multiple nodes is recommended.
     >>> startup_nodes = [{"host": "127.0.0.1", "port": "7000"}]
 
-    >>> # Note: decode_responses must be set to True when used with python3
+    >>> # Note: decode_responses gives different results with python3 and python2
     >>> rc = StrictRedisCluster(startup_nodes=startup_nodes, decode_responses=True)
 
     >>> rc.set("foo", "bar")
     True
     >>> print(rc.get("foo"))
     'bar'
 
+It is fairly safe to assume that any methods not documented here wil behave in the same way as
```

Comment:
Not sold on the working `...failry safe...`. Please give it some more consideration.